src/styles/_includes/_vars.scss

```
@@ -49,6 +49,7 @@ $gitmojis: (
   ok-hand: #c5e763,
   package: #fdd0ae,
   page-facing-up: #d9e3e8,
+  passport-control: #ffba54,
```

```suggestion
  passport-control: #4dc6dc,
```
(that's the :rocket: color)